src/main/java/org/takes/http/BkBasic.java

```diff
@@ -70,16 +84,20 @@ public BkBasic(final Take tks) {
         this.take = tks;
     }
 
+    // @todo #306 15min/DEV : the server should use same socket for new coming
```

Comment:
@mstzn36  `@todo #306:30min/DEV`  You missed `:`
please use javadoc-style for the comment
```
/**
*
*/
```